packages/ListBox/src/ListBox.js

```diff
@@ -54,10 +54,10 @@ export function ListBox(props) {
   return (
     <React.Fragment>
       {trigger}
-      <Content onCancelFooter={footer ? footer.props.onClickCancel : null}>
+      <Content onCancelFooter={footer ? footer.props.onClickCancel : null} hasOptions={React.Children.count(children)}>
         <Box {...box.props}>
           {filter}
-          <List height={height}>
+          <List height={height} hasOptions={React.Children.count(children)}>
```

Comment:
`hasOptions` is boolean, but you are passing in a number?  `!!React.Children.count(children)`?